### DISCUSSION TITLE
apt installing vs py installing mode

### DISCUSSION THREAD

**Discussion by peppelongo96** [nicolargo/glances#1901] [Q&A]
Hi. I installed successfully the glances module by using the provided "apt" command. Is there any difference respect to the procedure provided in the README here? I mean, this last one has a very long installation time compared to the apt installation.

**Comment by nicolargo** [comment]
Hi @peppelongo96 

the apt version is provided by the Debian community. If you want to use the latest Glances version please use the pip install method.

Nicolas
